src/treeffuser/_score_models/_neural_score_model.py

```diff
@@ -8,6 +8,10 @@
 from typing import Optional
 
 import numpy as np
+
+###################################################
+# Helper functions and classes
+###################################################
```

Comment:
can you just move that away (not in the middle of the imports?)

Reply:
Hahaha yesterday I thought this meant something else. but I just saw what was wrong lol